.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false  # Use container-based infrastructure
 language: python
 python:
   - "2.7"
-  - "3.4"
+  - "3.5"
```

Comment:
I don't know if it matters, I just did it to see if it changes the test results on Travis (which it doesn't). 

Maybe worth standardizing on a newer release, 3.5 seems to have pretty broad adoption.